config.go

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	optionPrefix      = "agent."
+	logLevelFlag      = optionPrefix + "log"
+	kernelCmdlineFile = "/proc/cmdline"
+)
+
+type agentConfig struct {
+	logLevel logrus.Level
```

Comment:
I assume we want to use some kind of wrapper for logrus/glog in the agent as well?